assertj-core/src/main/java/org/assertj/core/internal/Files.java

```diff
@@ -454,6 +458,24 @@ public void assertHasNoParent(AssertionInfo info, File actual) {
     throw failures.failure(info, shouldHaveNoParent(actual));
   }
 
+  public void assertIsEncodedIn(WritableAssertionInfo info, File actual, Charset charset, boolean lenient) {
```

Comment:
use `AssertionInfo` instead of `WritableAssertionInfo` as we are not changing it